packages/apollo-server-fastify/README.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Fastify
+description: Setting up Apollo Server with Fastify
+---
+
+[![npm version](https://badge.fury.io/js/apollo-server-core.svg)](https://badge.fury.io/js/apollo-server-core) [![Build Status](https://circleci.com/gh/apollographql/apollo-cache-control-js.svg?style=svg)](https://circleci.com/gh/apollographql/apollo-cache-control-js) [![Coverage Status](https://coveralls.io/repos/github/apollographql/apollo-server/badge.svg?branch=master)](https://coveralls.io/github/apollographql/apollo-server?branch=master) [![Get on Slack](https://img.shields.io/badge/slack-join-orange.svg)](https://www.apollographql.com/#slack)
```

Comment:
Aside from the fact that one of these badges is referring to `apollo-cache-control-js` (something which seems to be the case on the root `README.md` too and not something you've specifically introduced), let's not include these badges for now.